src/main/java/unrc/dose/User.java

```diff
@@ -142,4 +143,35 @@ private void load(final User u, final String name,
         u.set(ADMIN, admin);
     }
 
+	 /**
+     * this method remove logically a user.
+     * @param usernameUser this username is for delete logicaly,
+     * his account associate
+     * @param passwordUser this param is used for confirm the operation
+     * @return a boolean with this exit of remove or no remove the user
+     */
+	public static boolean deleteUser(final String usernameUser,
```

Comment:
The method's name should reflect the comments made above, that is, the user is not being deleted, his account is being disable, so `disableUser` or `setAsInactive` could be more appropriate names.

Method's arguments are a bit redundant, `usernameUser` and `passwordUser` shouldn't have `User` as part of their names.